include/sfx.h

```diff
@@ -229,6 +229,7 @@
 #define NA_SE_PL_DUMMY_221 0x8DD
 #define NA_SE_PL_SLIP_HEAVYBOOTS_LEVEL 0x8DE
 #define NA_SE_PL_SLIP_ICE_LEVEL 0x8DF
+// TODO: The sfxs below in this bank are not in gSoundParams and read OoB. May be fake Ids from OoT3D
 #define NA_SE_PL_JUMP_METAL 0x8E0
```

Comment:
minor thing that doesnt matter too much, but `sfxs` has always felt weird to me
sfx is already an abbreviation for "sound effects", just by how it sounds when sounding out each letter right. and context will tell you otherwise if its referring to one sound effect or not

just something to think about as sfx documentation gets further along i guess

Reply:
when i see sfxs i read "sound effectses", to give perspective on why it feels weird to me. but if other people dont care then its whatever to me lol

Reply:
Both sfx and sfxs feel fine to me, but I see where you're coming from. I'll change it to sfx then, it reads the same to me and its better for you